smart_card_connector_app/src/reader-tracker.js

```diff
@@ -20,6 +20,7 @@
  * NaCl (check readerfactory_nacl.cc for more information).
  */
 
+goog.provide('GoogleSmartCard.ReaderInfo')
```

Comment:
How do I make ReaderInfo structure local? (closure compiler forces me to put up this provide line)